consul/server.go

```diff
@@ -388,6 +390,9 @@ func (s *Server) setupSerf(conf *serf.Config, ch chan serf.Event, path string, w
 	if s.config.NonVoter {
 		conf.Tags["nonvoter"] = "1"
 	}
+	if s.config.VerifyIncoming {
+		conf.Tags["tls_verify_incoming"] = "1"
```

Comment:
Don't we need to key this off of certs being configured and have a sense more like `use_tls`? During the transition we will have to have verify turned off, but the server set up to handle incoming TLS connections.